test_runner/src/test/kotlin/ftl/args/IosArgsTest.kt

```diff
@@ -79,6 +79,7 @@ class IosArgsTest {
           num-flaky-test-attempts: 4
           type: xctest
           test-special-entitlements: true
+          
```

Comment:
Why this extra line is needed?

Reply:
absolutely no idea why this is here. Removing